### DISCUSSION TITLE
console 后台页面的文档 title 如何修改 每次都是动态切换

### DISCUSSION THREAD

**Discussion by Kimser** [halo-dev/halo#5539] [General]
### 系统信息

![image](https://github.com/halo-dev/halo/assets/54876002/08fa5ad3-013d-4e3f-92c4-1a658a68c0b7)

### 使用的哪种方式运行？

Docker

### 发生了什么？

![image](https://github.com/halo-dev/halo/assets/54876002/c8c6e441-b48b-4f83-8575-9e8733f0b107)

**Comment by guqing** [comment]
以后向这种问题请不要提到 issue 里面建议直接到论坛或者 discussion提问,issue 只接受建议和问题反馈
/kind support
